timev.rb

```diff
@@ -1,112 +1,124 @@
+#   Time.at(946702800, 500, :millisecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# _Microseconds_
+#
+# These forms accept an integer number of seconds +sec_i+,
+# a numeric number of microseconds +msec+,
+# an optional symbol +:usec+ or +:microsecond+,
+# and an optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800, 500000)                             # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, :usec)                      # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, :microsecond)               # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, in: '+09:00')               # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000, :usec, in: '+09:00')        # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000, :microsecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# _Nanoseconds_
+#
+# These forms accept an integer number of seconds +sec_i+,
+# a numeric number of nanoseconds +nsec+,
+# a symbol +:nsec+ or +:nanosecond+,
+# and an optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800, 500000000, :nsec)                     # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000000, :nanosecond)               # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000000, :nsec, in: '+09:00')       # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000000, :nanosecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# Parameters:
+# :include: doc/time/sec_i.rdoc
+# :include: doc/time/msec.rdoc
+# :include: doc/time/usec.rdoc
+# :include: doc/time/nsec.rdoc
+# :include: doc/time/tz.rdoc
+#
 def Time.at(time, subsec = (nosubsec = true), unit = (nounit = true), in: nil)
   __builtin.time_s_at(time, subsec, unit, __builtin.arg!(:in), nosubsec, nounit)
 end
 
 class Time
   # call-seq:
-  #    Time.new -> time
-  #    Time.new(year, month=nil, day=nil, hour=nil, min=nil, sec=nil, tz=nil) -> time
-  #    Time.new(year, month=nil, day=nil, hour=nil, min=nil, sec=nil, in: tz) -> time
+  #    Time.new -> new_time
+  #    Time.new(year, month=nil, day=nil, hour=nil, min=nil, sec=nil, tz=nil) -> new_time
+  #    Time.new(year, month=nil, day=nil, hour=nil, min=nil, sec=nil, in: tz) -> new_time
   #
-  # Returns a Time object.
+  # Returns a new \Time object based the on given arguments.
   #
-  # It is initialized to the current system time if no argument is given.
+  # In the first form (no arguments), returns the value of Time.now:
   #
-  # *Note:* The new object will use the resolution available on your
-  # system clock, and may include subsecond.
+  #   Time.new                                       # => 2021-04-24 17:27:46.0512465 -0500
   #
-  # If one or more arguments are specified, the time is initialized to the
-  # specified time.
+  # In the second form, argument +year+ is required and argument +tz+ is optional:
```

Comment:
All arguments other than `year` are optional, so it seems unnecessary to point out that `tz` is optional.  We should also use `zone` instead of `tz`, as that is what will be shown by `Time.instance_method(:initialize).parameters`.

Reply:
Done.